package.json

```diff
@@ -78,4 +79,4 @@
     "async": "^1.5.0",
     "npmi": "^1.0.1"
   }
-}
\ No newline at end of file
+}
```

Comment:
Why do we do the components in this PR, but have Travis do the main framework? Seems like we should be consistent.